website/docs/d/app_config_collection.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: 'App Configuration'
+layout: 'ibm'
+page_title: 'IBM : App Configuration collection'
+description: |-
+Get information about collection
```

Comment:
Give a space  and then add the description else res/datasoruce will be dispalyed outside the subcategory
https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/data-sources/app_config_environment (simialr to this) please fix them also along with this PR

Reply:
Addressed it